docs/installation/server/Installing_Basic_PKI_Server.adoc

```diff
@@ -2,7 +2,7 @@
 
 
 Follow this process to create and configure a basic PKI server without any of the PKI subsystems.
-This would be useful to troubleshoot general server issues (e.g. SSL).
+This would be useful to troubleshoot general server issues, for example `SSL`.
```

Comment:
I don't think we should use backticks here either since SSL is used as part of the description, not something that you would type or appears on the screen.